### ISSUE TITLE
Image generation results are easily affected by random seed

### ISSUE THREAD

**Issue by CodingMice** [ali-vilab/Ranni#14]
I got good results using the recommended prompt and seed, but when I used the recommended prompt and random seed I got worse results.
Good Result:
  input prompt: a black dog and a white cat
  input seed: 15
![a black dog and a white cat_1](https://github.com/ali-vilab/Ranni/assets/22537112/61b0d5fb-1dc6-4790-935b-41f5972cfb91)

Bad Result:
  input prompt: 10 apples on the table
  input seed: 15
![10 apples on the table_1](https://github.com/ali-vilab/Ranni/assets/22537112/151e7a5a-86aa-4b1f-9c74-1252817580b3)
Any solutions?

**Comment by thss15fyt** [comment]
Thanks for your interest.
1. It seems the output image is in wrong BGR instead of RGB. Please check the inference code.
2. The output results depend on both the text-to-panel and panel-to-image results. In your case, 10 apples are more diffcult for the first stage to properly arrange them. You can try different arrangement using random seed at first, and then run the panel-to-image with satisfying arrangement.

**Comment by holytony** [comment]
> I got good results using the recommended prompt and seed, but when I used the recommended prompt and random seed I got worse results. Good Result: input prompt: a black dog and a white cat input seed: 15 ![a black dog and a white cat_1](https://private-user-images.githubusercontent.com/22537112/322424237-61b0d5fb-1dc6-4790-935b-41f5972cfb91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMzg5MDUsIm5iZiI6MTcxMzMzODYwNSwicGF0aCI6Ii8yMjUzNzExMi8zMjI0MjQyMzctNjFiMGQ1ZmItMWRjNi00NzkwLTkzNWItNDFmNTk3MmNmYjkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDA3MjMyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg3YjY4OGU1ZGYyZTZkNzAzNGVjYmM0ODBlNGQ5ODlkZDA4YmQ4ZmJjMmM4ZTEzZDY5MjI1MDA0Y2ZmOTUzNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y28mk45mUJ05irblujYlcgo0RvZ6Q2WapA4jSpE0AC4)
> 
> Bad Result: input prompt: 10 apples on the table input seed: 15 ![10 apples on the table_1](https://private-user-images.githubusercontent.com/22537112/322423792-151e7a5a-86aa-4b1f-9c74-1252817580b3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMzg5MDUsIm5iZiI6MTcxMzMzODYwNSwicGF0aCI6Ii8yMjUzNzExMi8zMjI0MjM3OTItMTUxZTdhNWEtODZhYS00YjFmLTljNzQtMTI1MjgxNzU4MGIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDA3MjMyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwODMxY2NjNzZkYjQ1M2RkMWUzZmNiODI5ZmEwMTY2YjBkYThmMGU2MjZhMWNkYTA1YTE1YWZkMjhiM2I3NzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bNi_oLQ5x7uIxlQZoH1UNu9Nc03XKX7DqjdscM14uT4) Any solutions?

I agree, after having tried many times with different prompts and seeds, most of results coming out are not desirable,
generally speaking they still got issues with colours and numbers, I feel like the default demo examples are kinda of cherry picking, hopefully they can release the version they used in the original paper
